ideas/306-core-browser.md

```diff
@@ -0,0 +1,59 @@
+---
+id: #306-core-browser
+title: Core Browser
+status: implementation
+lead-contributor: Julien
+contributors:
+    - Julien
+    - Andrey
+    - Nastya
+    - Maciej
+    - Rachel
+budget:
+- actual: xxx
+- estimate: yyy
+- currency: ETH
+---
+
+Browser Swarm Proposal
+=
+
+## Summary and Goals
+
+The Core Browser team is going to be primarily responsible in Q12019 for building the P0 SNT Utility feature Stickers market.
+
+This swarm is also responsbile for maintaining and improving the overall browser experience. 
+
+Any capacity leftover from Stickers market tasks will be dedicated to essential browser features or fixes.
+
+## Contributors
+
+- Julien
+- Andrey
+- Nastya
+- Maciej
+- Rachel
+
+(+ Janitor swarm)
+
+## Communication
+
+`Status channel`: #status-core-dapps
+`Sync schedule`: currently twice weekly
+`Meeting notes`: Description :https://notes.status.im/wyjNCxeURu6lYSwWGh3biw
```

Comment:
Could you just add...

`Pivotal link`: https://www.pivotaltracker.com/projects/2231918

We'll use the same board as for stickers, and everything that's not stickers will be a lower priority.

Reply:
:+1: